packages/client/verifiable-credentials/src/verification/services/nftStatus.test.ts

```diff
@@ -0,0 +1,81 @@
+import { EVMNFT } from "@crossmint/common-sdk-base";
+
+import { NFTStatusService } from "./nftStatus";
+
+jest.mock("../../services/provider");
+
+describe("NFTStatusService", () => {
+    let service: NFTStatusService;
+
+    beforeEach(() => {
+        service = new NFTStatusService("testEnvironment");
+        jest.spyOn(service, "getNftOwnerByContractAddress").mockImplementation(async (contractAddress, tokenId) => {
+            if (tokenId === "notBurned") {
+                return "mockOwner";
+            }
+            if (tokenId === "burn error") {
+                throw new Error("ERC721: invalid token ID");
+            }
+            if (tokenId === "error") {
+                throw new Error("error");
+            }
+            throw new Error("ERC721: invalid token ID");
+        });
+    });
+
+    it("should return false if NFT is not burnt", async () => {
+        const mockNFT: EVMNFT = {
```

Comment:
what happens if the NFT is still minting?

Reply:
is not possible, you receive a credential only after the nft has been minted